app/misc.py

```diff
@@ -896,23 +896,26 @@ def getSinglePost(pid):
     return posts
 
 
-def postListQueryBase(*extra, nofilter=False, noAllFilter=False, noDetail=False, adminDetail=False):
+def postListQueryBase(*extra, nofilter=False, noAllFilter=False, noDetail=False, adminDetail=False, isSubMod=False):
     if current_user.is_authenticated and not noDetail:
         posts = SubPost.select(SubPost.nsfw, SubPost.content, SubPost.pid, SubPost.title, SubPost.posted,
                                SubPost.deleted, SubPost.score, SubPost.ptype,
                                SubPost.thumbnail, SubPost.link, User.name.alias('user'), Sub.name.alias('sub'),
                                SubPost.flair, SubPost.edited, Sub.sid,
                                SubPost.comments, SubPostVote.positive, User.uid, User.status.alias('userstatus'),
-                               *extra)
+                               *extra, *([SubPostReport.id.alias('open_report_id')] if isSubMod else [Value(None).alias('open_report_id')]), *([fn.Count(SubPostReport.id).alias('open_reports')] if isSubMod else [Value(None).alias('open_reports')]))
```

Comment:
This should be simplified to something like `*([SubPostReport.id.alias('open_report_id'), fn.Count(SubPostReport.id).alias('open_reports')] if isSubMod else [Value(None).alias('open_report_id'), Value(None).alias('open_reports')])`

Reply:
done and done!